git-hooks/pre-push

```diff
@@ -5,6 +5,36 @@ const path = require('path');
 const execa = require('execa');
 const chalk = require('chalk');
 
+const confirmIfMasterMessage = `${chalk.red(
+    `${chalk.inverse(
+        'You are about to push master!'
+    )}\nIs that what you intended?`
+)} [y/N]`;
+const confirmIfMaster = execa
+    .shell(
+        `
+        protected_branch='master'
+        current_branch=$(git symbolic-ref HEAD | sed -e 's,.*/\\(.*\\),\\1,')
```

Comment:
`git rev-parse --abbrev-ref HEAD` should be faster and more reabable

Reply:
oh nice one, cheers